problem_builder/v1/upgrade.py

```diff
@@ -37,14 +37,18 @@
 from lxml import etree
 from six import StringIO
 
-from courseware.models import StudentModule
 from mentoring import MentoringBlock
 from problem_builder.mentoring import MentoringBlock as NewMentoringBlock
 
+from .platform_dependencies import StudentModule
 from .studio_xml_utils import studio_update_from_node
 from .xml_changes import convert_xml_to_v2
 
 
+if not StudentModule:
+    raise ImportError("Could not import StudentModule from edx-platform courseware app.")
```

Comment:
Previous to this PR, `upgrade.py` would raise an ImportError if edx-platform is not importable.

`from .platform_dependencies import StudentModule`, though, will fail silently.

So, I added this if-statement to replicate the old error behavior.